source/includes/mongodb-compatibility-table-php.rst

```diff
@@ -7,50 +7,23 @@
      - MongoDB 8.0
      - MongoDB 7.0
      - MongoDB 6.0
-     - MongoDB 5.0
-     - MongoDB 4.4
-     - MongoDB 4.2
-     - MongoDB 4.0
-     - MongoDB 3.6
 
    * - ext + lib 1.21 to 2.0
      - ✓
      - ✓
      - ✓
-     - ✓
-     - ✓
-     - ✓
-     - ✓ [#deprecation-note]_
-     -
 
    * - ext + lib 1.20
```

Comment:
This can be combined with the section above now that the earlier versions are removed